.firebaserc

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
Currently this method doesn't work as intended on my local machine in terms of getting the Coming Soon Cards to display. Here are some screenshots as proof. 
![image](https://github.com/user-attachments/assets/68ed44d7-b2fd-4c4a-a1e7-90836538d63c)
![image](https://github.com/user-attachments/assets/9ec20b17-b9a3-4a37-a40a-026bc6215442)''

Building tags work as intended! But a new method must be implemented for coming soon cards